src/parser/jobs/pld/modules/Faith.tsx

```diff
@@ -0,0 +1,72 @@
+import {Plural, Trans} from '@lingui/react'
+import {DataLink} from 'components/ui/DbLink'
+import {StatusKey} from 'data/STATUSES'
+import {Event, Events} from 'event'
+import {Analyser} from 'parser/core/Analyser'
+import {EventHook} from 'parser/core/Dispatcher'
+import {filter, oneOf} from 'parser/core/filter'
+import {dependency} from 'parser/core/Injectable'
+import {Data} from 'parser/core/modules/Data'
+import Suggestions, {SEVERITY, TieredSuggestion} from 'parser/core/modules/Suggestions'
+import React from 'react'
+
+const TRACKED_STATUSES: StatusKey[] = [
+	'BLADE_OF_FAITH_READY',
+]
+
+export class Faith extends Analyser {
+	static override handle = 'faith'
+
+	@dependency private data!: Data
+	@dependency private suggestions!: Suggestions
+
+	private droppedFaiths: number = 0
+	private usedFaith: boolean = false
+
+	private faithHook?: EventHook<Events['action']>
+
+	override initialise() {
+		const trackedStatuses = TRACKED_STATUSES.map(key => this.data.statuses[key].id)
```

Comment:
I'd probably just assign this directly, since it's a custom analyser anyway, doesn't look like future changes to Faith would add more statuses.